src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -167,6 +176,11 @@ void setStats(ColibriStatsExtension stats)
             lastReportedBitrateKbps = bitrate;
         }
 
+        if (packetRateDown != null && packetRateUp != null)
```

Comment:
Yes. I think this might happen once when the bridge advertises stats for the first time, when only a small subset of them are included. All subsequent reports will have both